ckanext/switzerland/templates/scheming/display_snippets/fluent_tags.html

```diff
@@ -0,0 +1,13 @@
+<section class="tags">
```

Comment:
Overriding this template because the one in `ckanext-fluent` calls `h.scheming_language_text()` on the tag list, which is already a localized string at this point, not a dict.

(I don't know where this happens! It is very frustrating because if there are no tags for a language, it falls back to a different language, which is wrong, but I couldn't track it down.)

Reply:
the only thing I could imagine where it could happen (but possibly you have already investigated this) is in one of the default ckan-helpers, quite similar to what we had to monkey-patch.

But when overriding the template works after you have put in so much effort, I am fine with that :) 
And you explained it very well in your commit-message, too, that is great!